modules/programs/qutebrowser.nix

```
@@ -306,7 +306,17 @@ in {
       };
 
     xdg.configFile."qutebrowser/config.py" =
-      mkIf pkgs.stdenv.hostPlatform.isLinux { text = qutebrowserConfig; };
+      mkIf pkgs.stdenv.hostPlatform.isLinux {
+        text = qutebrowserConfig;
+        onChange = ''
+          hash="$(${pkgs.coreutils}/bin/echo -n $USER | ${pkgs.coreutils}/bin/md5sum | ${pkgs.coreutils}/bin/cut -d' ' -f1)"
```

```suggestion
          hash="$(echo -n \"$USER\" | ${pkgs.coreutils}/bin/md5sum | ${pkgs.coreutils}/bin/cut -d' ' -f1)"
```

The default shell used in scripts is defined by `pkgs.stdenv.shell` (bash globally).
AFAIK changing the default shell will incur a global rebuild.
This doesn't mean we can use and rely on bashism.

On a second note, popular shells like zsh, ksh, bash and fish all implement echo as a builtin command.